test/powershell/Host/HostUtilities.Tests.ps1

```diff
@@ -35,6 +35,12 @@ Describe "InvokeOnRunspace method as nested command" -tags "Feature" {
 Describe "InvokeOnRunspace method on remote runspace" -tags "Feature","RequireAdminOnWindows" {
 
     BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+
+        if (Test-IsWinWow64) {
```

Comment:
add a comment on why we are skipping

Reply:
We decided to skip remoting test on wow64 as it is too difficult to configure and we dont know if there is enough or any usage for this scenario